### PR TITLE
[DOC] use new surface API in plot_surf_atlas

### PR DESCRIPTION
**Pull request by Remi-Gau**
- Closes none but relates to #4568

Changes proposed in this pull request:

- part of moving surface API to stable codebase
- replace plot_surf_atlas with example taken from experimental

### CODE REVIEW

File: nilearn/experimental/surface/_datasets.py
Comment (Remi-Gau) [inline nilearn/experimental/surface/_datasets.py:121]: remove this "shortcut" to make it explicit in examples how we construct the SurfaceImage: [#4642](https://github.com/nilearn/nilearn/issues/4642)

**Review by bthirion — APPROVED** [review]
LGTM from looking at the changes, but the doc has not built yet ... to be checked...

**Comment by Remi-Gau** [comment]
blocked by #4674 
rebased after it is merged

File: doc/Makefile
Comment (bthirion) [inline doc/Makefile]: Is this related to this PR ?

**Review by bthirion — COMMENTED** [review]
LGTM overall !

File: doc/Makefile
Comment (Remi-Gau) [reply doc/Makefile]: nope: needed a faster way to build the doc locally (this does not generate the html report for maskers and glm , that usually takes some extra minutes)

**Pull request merged**
